python_modules/dagster/dagster/grpc/impl.py

```diff
@@ -67,7 +67,13 @@ def _report_run_failed_if_not_finished(instance, pipeline_run_id):
         yield instance.report_run_failed(pipeline_run)
 
 
-def core_execute_run(recon_pipeline, pipeline_run, instance, resume_from_failure=False):
+def core_execute_run(
+    recon_pipeline,
+    pipeline_run,
+    instance,
+    resume_from_failure=False,
+    set_exit_code_on_failure=False,
```

Comment:
im surprised we need to thread this through and change the `raise` behavior [1]

Reply:
oh this is `grpc` `core_execute_run` - I thought it was core core 

even still this isn't doing any exit code setting so feels like it should at least get renamed at this step

Reply:
`raise_on_reported_failure`  maybe ?